### DISCUSSION TITLE
× Unhandled Rejection (Error): register failed, invalid key request from plugin D:/upgrade-v4/web/src/app.ts.

### DISCUSSION THREAD

**Discussion by Khadhiri786** [umijs/umi#11903] [General]
![image](https://github.com/umijs/umi/assets/60467237/4cb3df50-89d8-4ba7-a536-eb65eb789baf)
![image](https://github.com/umijs/umi/assets/60467237/e21e1960-e5d7-4fbb-922f-456fe7691c3e)
I'm getting error while upgrading from umijs v3 to umijs v4

**Comment by fz6m** [comment]
if you are using umi-request (`request()` / `useRequest()`), need to enable the `request: {}` in config file:

```ts
// .umirc.ts (max)

  request: {}
```

```ts
// .umirc.ts (umi)

  plugins: ['@umijs/plugins/dist/request'],
  request: {}
```

**Reply by Khadhiri786** [reply]
its compiled, but I'm getting blank screen UI is not rendering
![image](https://github.com/umijs/umi/assets/60467237/2f2867cd-c65e-4941-a09a-2eaa9533a317)
![image](https://github.com/umijs/umi/assets/60467237/1507f6f2-4363-4fde-83e7-e7996689e142)

**Comment by Khadhiri786** [comment]
![image](https://github.com/umijs/umi/assets/60467237/3f0d4aea-e43e-421b-bf32-95f08a147b85)
I'm getting this error
